src/components/index.ts

```diff
@@ -1,3 +1,6 @@
 export * from './button';
 export * from './landing-block';
 export * from './icon';
+export * from './tab';
```

Comment:
for one component one export, even you exporting many components regarding your component